app/sentry.ts

```diff
@@ -0,0 +1,67 @@
+import * as Sentry from '@sentry/react';
```

Comment:
See the second point in the main comment

```suggestion
import * as Sentry from '@sentry/react';
import { Severity } from '@sentry/types';
```